pkg/hubble/filters/httpheader_test.go

```diff
@@ -0,0 +1,337 @@
+package filters
+
+import (
+	"context"
+	"testing"
+
+	flowpb "github.com/cilium/cilium/api/v1/flow"
+	v1 "github.com/cilium/cilium/pkg/hubble/api/v1"
+	"github.com/cilium/cilium/pkg/monitor/api"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestHTTPHeaderFilter(t *testing.T) {
+
+	type args struct {
+		f  []*flowpb.FlowFilter
+		ev []*v1.Event
```

Comment:
this is causing a compilation failure, as the declared args `ev` are `*v1.Event` now
```suggestion
		ev *v1.Event
```